src/containers/EventCreationPage.tsx

```diff
@@ -140,7 +195,9 @@ const EventCreationPage = () => {
     <div className="landing-wrapper">
       <div className="event-creation-top-section">
         <div className="landing-top-bar">
-          <Typography variant="h3">Create New Event</Typography>
+          <Typography variant="h3">
+            {mode === 'new' ? 'Create New Event' : 'Edit Event'}
+          </Typography>
```

Comment:
On the (new) line 235, you need to update the logic since the button shouldn't say "Create" if `mode === 'edit'`

Also would suggest putting `'new'` and `'edit'` into an enum somewhere.

![image](https://user-images.githubusercontent.com/22457859/90340400-e878be80-dfc5-11ea-93ed-2b4480ed8e43.png)

Reply:
Good catch, forgot to include that. I'll also move 'new' and 'edit' to an enum